packages/cloudvision-connector/src/Connector.ts

```diff
@@ -169,12 +171,25 @@ export default class Connector extends Wrpc {
     return this.get(GET_DATASETS, params, makeNotifCallback(callback));
   }
 
+  /**
+   * Returns a list of configs (datasets with type == 'config').
+   */
+  public getConfigs(callback: NotifCallback): string | null {
+    const configType: typeof CONFIG_DATASET_TYPE[] = [CONFIG_DATASET_TYPE];
+    const params: ConfigParams = { types: configType };
+    return this.get(GET_DATASETS, params, makeNotifCallback(callback));
+  }
+
   /**
    * Returns a list of datasets. This will return data sets of type 'device',
-   * as well as type 'app'.
+   * as well as type 'app' & type 'config'.
```

Comment:
Reword to better reflect at dataset types are a list of n types

Reply:
Done